lib/client/setClientHeader.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+function setClientHeader(method,model,options){
+    options.headers = options.headers || {};
+    options.headers['Brisket-Client'] = true;
```

Comment:
I think we use lower case below. how about lower case everywhere for consistency?